tests/e2e/conftest.py

```diff
@@ -65,6 +88,24 @@ def get_secretkey(request):
         pytest.fail("--secretkey is required")
     return secret_key
 
+def get_installation_option(request):
+    installation_option = request.config.getoption("--installation_option")
+    if not installation_option:
+        installation_option = "kustomize"
+    return installation_option
+
+def get_depployment_option(request):
+    deployment_option = request.config.getoption("--deployment_option")
+    if not deployment_option:
+        deployment_option = "vanilla"
+    return deployment_option
+
+def get_aws_telemetry_option(request):
+    aws_telemetry_option = request.config.getoption("--aws_telemetry")
+    if not aws_telemetry_option:
+        aws_telemetry_option = "enable"
+    return aws_telemetry_option
+
```

Comment:
why is this needed as a separate option? what testing purpose does it serve?

Reply:
I think you are right, it's needed for the kubeflow installation script, but we can remove it in the test